src/main/scala/piconot/Empty.scala

```diff
@@ -0,0 +1,32 @@
+package piconot
+
+/**
+ * @author aputman
+ */
+object Empty extends PracticalBot {
+  picobot("empty") (
+    state ("toCorner") (
+      move(left),
+      move (up) (
+        check(left -> blocked)
+      ),
+      stay (
+        check(left -> blocked, up -> blocked) -> "toRight"
+      )
+    ),
+    
+    state ("toRight") (
+      move (right),
+      move (down) (
+        check(right -> blocked) -> "toLeft"
+      )
+    ),
+    
+    state ("toLeft") (
+      move (left),
+      move (down) (
```

Comment:
I feel like this syntax implies that the state first moves left, then moves down